src/functions-and-arrays.js

```diff
@@ -1,18 +1,84 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(x,y) {
+  if (x>=y) return x;
+  if (x<y) return y;
+}
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+function findLongestWord(wordArr) {
+  let l=0;
+  let longestWordIndex=0;
+  for (let i=0; i<wordArr.length; i++) {
+    if (wordArr[i].length>l) {
+      l=wordArr[i].length;
+      longestWordIndex=i;
+    }
+  }
+  return wordArr[longestWordIndex];
+}
```

Comment:
to solve this error
`should return null if receives an empty array when called`
you can just check if the wordArr that you are receiving in the function is empty
```
if (numbersAvg.length === 0) return null;
```